api/docs/guides/test.md

```diff
@@ -12,12 +12,3 @@
    ```bash
    just api-test
    ```
-
-## On the host
-
-1. Start the Django dev server. See the {doc}`"Run" <./run>` document for instructions.
-
-2. Run the tests in a terminal on the host.
-   ```bash
-   just api-testlocal
-   ```
```

Comment:
Why is this removed?

Reply:
Since I hadn't mapped the DBs and Redis to the host, testing locally was not possible. Now that you and @AetherUnbound have expressed support for mapping these services locally, we can restore this functionality.

Reply:
I typically use `api-testlocal` when running unit tests locally.

Reply:
Can we fix that in a separate PR? Our tests are fundamentally flawed in that there is no separation between unit tests which should run in the Django environment and E2E/integration tests which should run outside. This means that when ports are different  inside and outside the container they cannot use the same env vars.

Updating tests would make this PR quite big and hard to review. So I'm hoping we can open a high-priority issue for that and fix it in post.

As a workaround, you can speed up test execution by running specific tests by passing a filename after `just api-test <file_name>`.

Reply:
I always run specific tests when writing new ones, the issue is the container startup time and difficulties with attaching debuggers.

I'm fine solving it in a follow up PR but please open an issue describing the problem and the solution. I'm not entirely convinced the integration tests need to run outside the container (see Playwright tests in the frontend, for example) but we can discuss the trade offs in another issue as you suggested.

Reply:
FWIW the catalog has a just recipe for setting up a "test session" `docker run`, which I use frequently for rapidly iterating on tests: 
https://github.com/WordPress/openverse-catalog/blob/40b4306b161bb4328f4b2e77bc9790e34332eb41/justfile#L65-L67

Perhaps it could be used here!